### PR TITLE
[Insights] Geo Map insights documentation added

### PR DESCRIPTION
**Pull request by yomalip**
## Purpose
Added documentation for new Insights component "Geo Map"

## Related PRs
https://github.com/wso2-enterprise/choreo-console/pull/5676
https://github.com/wso2-enterprise/choreo-apim-analytics-portal/pull/495

### CODE REVIEW

File: en/docs/insights/view-api-insights.md [hunk 1]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
 
 To download a pregenerated report, click **Download** for the relevant month and then click on the required format.
 
+### Geo Map
+
+The **Geo Map** page allows you to see a geographical representation of your API usage. You can use this page to make business decisions such as identifying growing regions, analyzing the impact of a region specific marketing campaign, etc.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:286]:
```suggestion
The **Geo Map** page allows you to see a geographical representation of your API usage. You can use this page to make business decisions such as identifying growing regions, analyzing the impact of a region-specific marketing campaign, etc.
```

File: en/docs/insights/view-api-insights.md [hunk 2]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
...
+
+![Geo Map page](../assets/img/insights/geomap-page-full.png){.cInlineImage-full}
+
+To filter the information displayed on this page by API, select the required API(s) in the **API** field. **All** is selected by default.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:290]:
```suggestion
To filter the information displayed on this page by API, select the required API(s) in the **API** field. The default value is **All**.
```

Comment (RukshiW) [reply en/docs/insights/view-api-insights.md:290]: Suggesting this change to avoid passive voice.

File: en/docs/insights/view-api-insights.md [hunk 3]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
...
 
+### Geo Map
+
+The **Geo Map** page allows you to see a geographical representation of your API usage. You can use this page to make business decisions such as identifying growing regions, analyzing the impact of a region-specific marketing campaign, etc.
+
+![Geo Map page](../assets/img/insights/geomap-page-full.png){.cInlineImage-full}
+
+To filter the information displayed on this page by API, select the required API(s) in the **API** field. The default value is **All**.
+
+This heat map highlights the countries with the appropriate color measured by relative color scale measured based on highest hit count to lowest non-zero hit count.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:292]:
```suggestion
This heat map highlights the countries with the appropriate color based on the hit count. It uses a relative color scale to depict the highest hit count to the lowest non-zero hit count.
```

File: en/docs/insights/view-api-insights.md [hunk 4]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
...
+
+![Geo Map Hit Scale](../assets/img/insights/geomap-hit-count-scale.png){.cInlineImage-half}
+
+#### Viewing number of requests by country
+
+You can hover over the highlighted countries to view the country name and the total hit counts originated from that location. The map can be zoomed and panned using mouse gestures.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:298]:
```suggestion
You can hold the pointer over the highlighted countries to view the country name and the total number of hits from that location. You can also use the mouse to zoom the map or pan it.
```

File: en/docs/insights/view-api-insights.md [hunk 5]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
...
+
+![Geo Map by country](../assets/img/insights/geomap-requests-by-country.png){.cInlineImage-half}
+
+!!! Note
+    Currently, this feature is only available for On Premise environments.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:303]:
``` suggestion
    Currently, this feature is only available for on-premise environments.

File: en/docs/insights/view-api-insights.md [hunk 6]
```diff
@@ -281,3 +281,25 @@ These widgets allow you to download monthly reports for the last three months.
...
+
+Optionally the geo map data can be downloaded in a tabular format using the PDF or CSV download options using ![download](../assets/img/insights/insights-download-report.svg) on top right corner.
```
Comment (RukshiW) [inline en/docs/insights/view-api-insights.md:305]:
```suggestion
Optionally you can download geo map data in a tabular format using the PDF or CSV download options by clicking ![download](../assets/img/insights/insights-download-report.svg) on the top right corner.
```

**Review by fazlan-nazeem — APPROVED** [review]

**Review by RukshiW — APPROVED** [review]

**Review by RukshiW — APPROVED** [review]

**Pull request merged**